docs/docs/index.md

```diff
@@ -48,9 +48,21 @@ Let's look at an example.
 
 Imagine you work for an organization with the following system:
 
-<!--- source: https://docs.google.com/drawings/d/1nPZ08-G8zEZyTlVTmLGnlnS-CdBEltiV8kds0cwd_qQ/edit --->
+```mermaid
+graph
+    nX["Public Network X"] --> Internet
+    nY["Private Network Y"]
 
-![Example System](system.svg)
+    pW["Port W"] --> nX
+    pX["Port X"] --> nX
+    pY["Port Y"] --> nY
+    pZ["Port Z"] --> nY
+
+    sX["Server X"] --> pW
+    sY["Server Y"] --> pX
+    sY --> pY
+    sZ["Server Z"] --> pZ
+```
```

Comment:
I guess an alternative could be something like

```mermaid
graph
    nX["Public Network X"] --> Internet
    nY["Private Network Y"]
    sX["Server X"] --"Port X"--> nX
    sY["Server Y"] --"Port X"--> nX
    sY --"Port Y"--> nY
    sZ["Server Z"] --"Port Z"--> nY
```

But this is fine, too. I think both are fine. 😄

Reply:
Thanks, that looks better to me as well so have used this now!